python2-pybind11/setup.py

```diff
@@ -3,7 +3,7 @@
 from distutils.core import setup, Extension
 from distutils import sysconfig
 
-cpp_args = ['-std=c++14', '-stdlib=libc++', '-mmacosx-version-min=10.7']
```

Comment:
Good to know.  Right now we're mostly building on gcc 4.8, so this usually isn't an option for us yet (and I don't think it's worth the effort of dealing with compiler-dependent options in distutils for just this example package).  I'll take a look at adding visibility hints to this package; it's more important that the code here be readable than the binaries be small, but I'd also like to demonstrate best-practice usage if I can.